samples/creation.ts

```diff
@@ -202,6 +202,15 @@ export async function run() {
             console.log(`found ${runs.length} test runs`);
         }
 
+        /********** TestPlan **********/
+        printSectionStart('TestPlan');
+        const testPlanApi = await vstsCollectionLevel.getTestPlanApi();
+        const testPlans: TestRun[] = await testPlanApi.getTestPlans(common.getProject());
```

Comment:
Should we handle NRE here , incase testPlanApi is null?

Reply:
Not required, this is just a sample example of how to use the new client. Also, testPlanApi  won't be null